### PR TITLE
fix(index): update examples according to API changes

### PR DESCRIPTION
**Pull request by cburgdorf**
r? @Ryman

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -150,16 +130,12 @@
...
 
-    fn post_handler (request: &Request, response: &mut Response) { 
-        let person = request.json_as::<&zwnj;Person>().unwrap();
-        let text = format!("Hello {} {}", person.firstname, person.lastname);
-        response.send(text.as_slice()); 
-    }
-    
-    server.post("/a/post/request", post_handler);
+    server.post("/a/post/request", middleware! { |request, response|
+        let person = request.json_as::&zwnj;Person>().unwrap();
```
Comment (Ryman) [inline index.html]: `json_as::<`

Comment (cburgdorf) [inline index.html]: We need to use html entities for it to work but I guess it should actually be a &lt; instead of an &zwnj;

Comment (Ryman) [inline index.html]: yeah, just pointing out that it's missing :P

Comment (cburgdorf) [inline index.html]: Fixed and merged ;)

**Pull request closed without merging**